src/OpenLoco/src/GameState.h

```diff
@@ -11,16 +11,25 @@
 #include "ScenarioObjective.h"
 #include "Station.h"
 #include "Town.h"
+#include <OpenLoco/Core/EnumFlags.hpp>
 
 namespace OpenLoco
 {
+    enum class GameStateFlags : uint32_t
+    {
+        none = 0U,
+        tileManagerLoaded = (1U << 0),
+        unk2 = (1U << 1),
+        preferredOwnerName = (1U << 2),
+    };
+    OPENLOCO_ENABLE_ENUM_OPERATORS(GameStateFlags);
+
```

Comment:
Worth putting these in their own unit/header file? So e.g. Audio.cpp doesn't have to include the whole GameState header. (cc @duncanspumpkin)

Reply:
Yes i was just thinking that as well.